cluster/examples/kubernetes/ceph/object-multisite-pull-realm.yaml

```
@@ -0,0 +1,64 @@
+#################################################################################################################
+# Create an object store with settings for a test environment. Only a single OSD is required in this example.
+#  kubectl create -f object-multisite.yaml
+#################################################################################################################
+apiVersion: v1
+data:
+  # these keys should be the base64 encoded versions of the actual keys or copied from the realm's on the other cluster's secret
+  access-key: VzFjNFltMVdWRTFJWWxZelZWQT0=
```

We need a really obvious comment about replacing these keys, perhaps with a TODO
```suggestion
  # TODO: Replace with keys for your cluster
  access-key: VzFjNFltMVdWRTFJWWxZelZWQT0=
```